### PR TITLE
Add full stops and capitals to exception messages in power board.

### LINKED ISSUE #496: Typo in Power Board Exception
https://github.com/j5api/j5/blob/4347e5bb3e573748f3b7387a640bcb382226af68/j5/backends/hardware/sr/v4/power_board.py#L127

### PR DESCRIPTION
**Pull request by trickeydan**
Fixes #496

### CODE REVIEW

File: j5/backends/hardware/sr/v4/power_board.py [hunk 1]
```diff
@@ -116,7 +116,7 @@ def set_power_output_enabled(
             cmd = CMD_WRITE_OUTPUT[identifier]
         except KeyError:
             raise ValueError(f"Invalid power output identifier {identifier!r}; "
```
Comment (tpoomlmly) [inline j5/backends/hardware/sr/v4/power_board.py]: Line 121 is the culprit that's failing the tests. I suggest moving the string onto a new line, although that will reduce consistency.
```suggestion
            raise ValueError(
                f"Invalid power output identifier {identifier!r}; "
                f"valid identifiers are {CMD_WRITE_OUTPUT.keys()}."
            ) from None
```

Comment (tpoomlmly) [reply j5/backends/hardware/sr/v4/power_board.py]: There's also no full stop on line 109, but adding one will push it over 90 characters long.

Comment (tpoomlmly) [inline j5/backends/hardware/sr/v4/power_board.py]: Line 119 is the culprit that's failing the tests. I propose putting the string on a new line, although that will reduce consistency.
```suggestion
            raise ValueError(
                f"Invalid power output identifier {identifier!r}; "
                f"valid identifiers are {CMD_WRITE_OUTPUT.keys()}."
            ) from None
```

Comment (tpoomlmly) [reply j5/backends/hardware/sr/v4/power_board.py]: There's also a missing full stop on line 109, but adding it will push the line over 90 characters.

Comment (trickeydan) [reply j5/backends/hardware/sr/v4/power_board.py]: > although that will reduce consistency.

We don't mind as long as it passes the linter :)

**Pull request merged**